pkg/fakerp/vault.go

```diff
@@ -75,12 +75,39 @@ func writeTLSCertsToVault(ctx context.Context, kvc azureclient.KeyVaultClient, c
 	return nil
 }
 
-func createVault(ctx context.Context, vc azureclient.VaultMgmtClient, appObjectID, tenantID, resourceGroup, location, vaultName string) (string, error) {
+func createVault(ctx context.Context, vc azureclient.VaultMgmtClient, writeObjectID, readObjectID, tenantID, resourceGroup, location, vaultName string) (string, error) {
```

Comment:
`(fakerp|rp|plugin)ObjectID, masterObjectID` clearer? (throughout PR)

Reply:
oops, missed this..